modules/admin_manual/pages/installation/docker/index.adoc

```diff
@@ -271,7 +305,7 @@ you can check the update log with the following command:
 
 [source,console]
 ----
-sudo docker-compose logs --timestamp owncloud
+docker-compose logs --timestamp owncloud
 ----
 
 === Docker Compose YAML File
```

Comment:
There is no statement about why and what to do with the yaml file. Some explaining words may help.

I am not a yaml knowledged person, but I see varaibles down there like `${...}`.
Some seem to be used like ${OWNCLOUD_VERSION} some set like ${HTTP_PORT}?
```
services:
  owncloud:
    image: owncloud/server:${OWNCLOUD_VERSION}
    container_name: owncloud_server
    restart: always
    ports:
      - ${HTTP_PORT}:8080
```

Reply:
For now only a minimum of explanation.